apps/els_lsp/src/els_code_lens.erl

```diff
@@ -8,12 +8,13 @@
 %% Callback Functions
 %%==============================================================================
 
--callback command(poi()) -> els_command:command_id().
--callback command_args(els_dt_document:item(), poi()) -> [any()].
+-callback init(els_dt_document:item()) -> state().
```

Comment:
This new, optional callback gives us a chance to run `TypEr` only once per file.